config/initializers/inflections.rb

```diff
@@ -34,6 +36,8 @@
   inflect.acronym 'SM'
   inflect.acronym 'SSOe'
   inflect.acronym 'SSO'
+  inflect.acronym 'UTC'
+  inflect.acronym 'Utc' # Utc is used in some places, UTC in others
```

Comment:
Won't `Utc` override `UTC`? Have you tested to make sure this works? Can you put some testing output in the PR description?

Reply:
Yeah you were right, this second inflection fixed test failures but overrode the first one. I took a different approach and added testing steps to the description